kuksa-client/setup.cfg

```diff
@@ -11,7 +11,7 @@ project_urls=
   Bug Tracker=https://github.com/eclipse/kuksa.val/issues
 classifiers =
     Intended Audience :: Developers
-    Development Status :: 3 - Alpha
+    Development Status :: 5 - Production/Stable
```

Comment:
This can be discussed, but I would say kuksa-client has passed alpha stage

https://pypi.org/classifiers/
https://martin-thoma.com/software-development-stages/

Reply:
would agree with production/stable

Reply:
Works for me. We might have a long argument about whether it is 4 or 5, but anyway, this is one of our few main/core components and it has been a long time around and is reasonably stable.